swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category=None, condition=0):
+        if not category:
+            category = ""
+        self.category = category
+        self.condition = condition
+    
+    def __str__(self):
+        return "Hello World!"
+    
+    def condition_description(self):
+        CONDITION_RATING = {
+            1: "parts only",
+            2: "play condition",
+            3: "good used condition",
+            4: "excellent used condition",
+            5: "like new condition"
+        }
+
+        condition = CONDITION_RATING[self.condition]
```

Comment:
This is such a great catch, Kelsey! My first thought would be to change line 20 to `condition = CONDITION_RATING[int(self.condition)]`, which would round the float down to the nearest integer. In this application I think rounding down makes the most sense since it's better to underestimate the condition of one's item versus overestimate.

Two other ideas that I thought of but seem less ideal:
1. Modifying the Item class to only allow integer values for condition.
2. Adding floats as keys in the CONDITION_RATING dictionary, although the issue then is managing the nearly infinite number of floats between each integer.

Reply:
Great consideration of approaches! I agree with your conclusions, it makes a lot of sense for this application to use `int()` to chop off the decimal part of the number. We could use `round()` instead to get the _closest_ integer to a decimal, but like you said, it's better to underestimate the condition of one's item versus overestimate!